### ISSUE TITLE
Current README.md contains a scam link (URL for `cmder` in `Credits` section)

### ISSUE THREAD

**Issue by KRACK-BIT** [mtgatracker/mtgatracker#561]
TL;DR - I think the current [Credits section](https://github.com/mtgatracker/mtgatracker/blob/d51f545807f0902d3a987ee0259ff2097154cbd6/README.md#credits-license) accidently links to a [scam](https://cmder.net/) instead of the [real deal](https://cmder.app/)

---

Was reading the README.md to learn about this project, and noticed that in the [Credits Section](https://github.com/mtgatracker/mtgatracker/blob/d51f545807f0902d3a987ee0259ff2097154cbd6/README.md#credits-license) mentioned a tool (`cmder`) I hadn't used before. However, the [current link](https://cmder.net/) goes to a website that does not look like a dev tool *at all*...

![image](https://github.com/user-attachments/assets/feb4c7d9-d441-451b-8962-a9f04b9f59f5)

A quick google search got me to what I think was the [intended link](https://cmder.app/) and its [GitHub repository](https://github.com/cmderdev/cmder)
![image](https://github.com/user-attachments/assets/543381e1-c2ef-43c6-90b9-46b9be7a91c2)

If this was an error, seems like a quick and easy thing to fix!
